### PR TITLE
Using JsonResponse for all json-like responses

### PR DESCRIPTION
**Pull request by jotes**
I decided to split this big refactoring into smaller steps. @Osmose @mathjazz could you review this again?

### CODE REVIEW

File: pontoon/base/views.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     HttpResponse,
     HttpResponseBadRequest,
     HttpResponseForbidden,
+    JsonResponse
```
Comment (Osmose) [inline pontoon/base/views.py]: Holy crap is this in Django now

File: pontoon/intro [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 30273cd602cb4ad2dbe669417769a9c17452f597
+Subproject commit dafff9cfab416bde2ffc6339e5ef0fc453811f99
```
Comment (Osmose) [inline pontoon/intro]: Seems a bit unrelated?

**Comment by Osmose** [comment]
r+, I'd like the submodule update to be removed but it won't harm anything if it's included, I think. Nice work!

**Comment by jotes** [comment]
@Osmose i reverted changed to pontoon-intro, could you look again? Thx.

**Comment by Osmose** [comment]
r+, leaving to @mathjazz to merge :D
